Source/Core/VideoCommon/TextureCacheBackendInterface.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "VideoCommon/BPMemory.h"
```

Comment:
`<cstddef>`, `<memory>`, `<tuple>`, and `Common/CommonTypes.h` should be included.